.dockerignore

```diff
@@ -2,9 +2,9 @@
 .github
 .gitignore
 data
-deploy_configs
-deploy_configs.tgz
 icubam.egg-info
 docker/configs/certbot
 docker/scripts
-docker/resources
\ No newline at end of file
+docker/resources
+test.db
+icubam.db
```

Comment:
Maybe more generally, `*.db` unless we can't use whildcards in `.dockerignore` ?

Reply:
Indeed. replaced with globstar.